lib/cn/kblock_builder.c

```diff
@@ -1150,6 +1149,21 @@ kbb_set_merge_stats(struct kblock_builder *bld, struct cn_merge_stats *stats)
     bld->mstats = stats;
 }
 
+bool
+kbb_is_empty(struct kblock_builder *bld)
```

Comment:
I think this should be renamed to reflect that it is referring to the current kblock similar to `kbb_curr_kblk_minmax_keys()`. Though I don't love the name so much, the intention is obvious.

Reply:
kbb_is_empty() checks whether there are any keys in the kblock builder, not just the current kblock.

If curr_kblock.num_keys == 0 then it implies that there are no keys in this kbb (we discussed about this in your PR comments).